models/src/main/scala/coop/rchain/models/rholangn/parmanager/Sorting.scala

```diff
@@ -0,0 +1,21 @@
+package coop.rchain.models.rholangn.parmanager
+
+import coop.rchain.models.rholangn._
+
+import java.util
+import scala.math.Ordered.orderingToOrdered
+
+private[parmanager] object Sorting {
+  implicit val o: Ordering[Array[Byte]] = (a: Array[Byte], b: Array[Byte]) =>
+    util.Arrays.compare(a, b)
+
+  def sortPars(ps: Seq[ParN]): Seq[ParN] = ps.sorted(Ordering.by((p: ParN) => p.rhoHash))
+  def sortBinds(bs: Seq[ReceiveBindN]): Seq[ReceiveBindN] =
+    bs.sorted(Ordering.by((b: ReceiveBindN) => b.rhoHash))
+  def sortBindsWithT[T](bs: Seq[(ReceiveBindN, T)]): Seq[(ReceiveBindN, T)] =
+    bs.sortBy(_._1.rhoHash)
```

Comment:
Try to use pattern matching more because it is more verbal about what you are actually accessing. Indices do not tell much and harder to read.
```scala
bs.sortBy { case (receiveBind, _) -> receiveBind.hash }
```